app/controllers/tag_controller.rb

```diff
@@ -246,7 +246,8 @@ def widget
   end
 
   def blog
-    nids = Tag.find_nodes_by_type(params[:id], 'note', 20).collect(&:nid)
+    blog_count = Tag.tagged_node_count('blog')
```

Comment:
So sorry I missed this! I think we can actually skip this line and specify a length of `nil` -- that'll remove the limit!